pkg/storage/merge_queue.go

```diff
@@ -123,6 +124,25 @@ func (mq *mergeQueue) enabled() bool {
 	return st.Version.IsMinSupported(cluster.VersionRangeMerges) && storagebase.MergeQueueEnabled.Get(&st.SV)
 }
 
+func (mq *mergeQueue) mergesDisabledForRange(desc *roachpb.RangeDescriptor) bool {
+	_, tableID, err := keys.DecodeTablePrefix(desc.StartKey.AsRawKey())
```

Comment:
Couldn't the range span multiple tables? I think this means you have to look at the range tableID1..tableID2